README.md

```diff
@@ -57,6 +57,9 @@ Let's take Unity Catalog for spin. In this guide, we are going to do the followi
   An example project is provided to demonstrate how to use the UC SDK for various assets
   as well as provide a convenient way to explore the content of any UC server implementation.
 
+> If you prefer to run Unity Catalog in Docker use `docker
+> compose up`. See the [Docker Compose docs](./docs/docker_compose.md) for more details.
+
```

Comment:
[1 / 2]

@dennyglee I was thinking this would be the ingress from the main docs to the Docker Compose instructions, which then contain information explaining how the UI is also started with `docker compose up`.

See next comment for more details :)

Reply:
Yes, I like where you are going with this, eh?!